src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerEventHubsRequestExtensions.cs

```diff
@@ -112,22 +113,23 @@ public static partial class ILoggerExtensions
         /// <param name="eventHubsName">The name of the Event Hub that the processor is connected to, specific to the EventHubs namespace that contains it.</param>
         /// <param name="operationName">The optional logical name that can be used to identify the operation that consumes the message.</param>
         /// <param name="isSuccessful">The indication whether or not the Azure EventHubs request was successfully processed.</param>
-        /// <param name="duration">The duration it took to process the Azure EventHubs request.</param>
         /// <param name="startTime">The time when the request was received.</param>
+        /// <param name="duration">The duration it took to process the Azure EventHubs request.</param>
         /// <param name="context">The telemetry context that provides more insights on the Azure EventHubs request.</param>
         /// <exception cref="ArgumentNullException">Thrown when the <paramref name="logger"/> is <c>null</c>.</exception>
         /// <exception cref="ArgumentException">
         ///     Thrown when the <paramref name="eventHubsNamespace"/>, <paramref name="consumerGroup"/>, <paramref name="eventHubsName"/> is blank.
         /// </exception>
+        /// <exception cref="ArgumentOutOfRangeException">Thrown when the <paramref name="duration"/> is a negative time range.</exception>
         public static void LogEventHubsRequest(
             this ILogger logger,
             string eventHubsNamespace,
             string consumerGroup,
             string eventHubsName,
             string operationName,
             bool isSuccessful,
-            TimeSpan duration,
             DateTimeOffset startTime,
+            TimeSpan duration,
```

Comment:
Updated the signature to be in line with the other extensions.
We can still do this as it's not yet released.